config/locales/pl/about.pl.yml

```diff
@@ -0,0 +1,22 @@
+pl:
+  about:
+    title: 'O REFUGE'
+    p1header: 'Czym jest REFUGE Restrooms?'
+    p1:
+      first: 'Refuge Restrooms to aplikacja u�atwiaj�ca osobom transp�ciowym i interp�ciowym bezpieczny dost�p do �azienek publicznych. U�ytkownicy mog� znale�� �azienki znajduj�ce si� w promieniu konkretnej lokalizacji, doda� now� �azienk� do listy, a tak�e skomentowa� i oceni� te dodane ju� wcze�niej.'
+      second: 'Naszym celem jest nie tylko znajdowanie obecnie istniej�cych ju� �azienek bezpiecznych dla os�b transp�ciowych, ale tak�e propagowanie dzia�a� maj�cych na celu zwi�kszenie bezpiecze�stwa os�b transp�ciowych i interp�ciowych.'
+    p2header: 'Sk�d pochodz� wszystkie dane?'
+    p2: 'Pierwsze 4500 lokalizacji pochodzi z danych nieprowadzonego ju� projektu Safe2Pee. Pozosta�e s� dodawane przez u�ytkownik�w. Je�eli zauwa�ysz, �e �azienka w miejscu publicznym jest neutralna, dodaj j� do bazy danych.'
+    p3header: 'Sk�d nazwa REFUGE (ang. azyl)?'
+    p3: 'Chcemy, �eby nasi u�ytkownicy byli godnie traktowani: bezpieczny dost�p do �azienki do podstawowe prawo cz�owieka. Chcemy zapewni� to bezpiecze�stwo ka�demu w razie potrzeby.'
```

Comment:
Może zostawić tytuł "Skąd nazwa REFUGE?" i tłumaczenie dodać do `p3`? To tylko sugestia stylistyczna.